### PR TITLE
Update paginated-reports-report-builder-power-bi.md

### PR DESCRIPTION
**Pull request by gsuttie**

### CODE REVIEW

**Comment by PRMerger12** [comment]
@gsuttie : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
@maggiesMSFT The update is not grammatically correct, but could you take a look? Thanks.

File: powerbi-docs/paginated-reports/paginated-reports-report-builder-power-bi.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "What are paginated reports in Power BI Premium?"
-description: Paginated reports, long the standard report format in SQL Server Reporting Services, are now available in the Power BI service. These reports can be printed or shared. You can control the report layout exactly. They display all the data in a table, for example, even if the table spans multiple pages.  
+description: Paginated reports, long been the standard report format in SQL Server Reporting Services, are now available in the Power BI service. These reports can be printed or shared. You can control the report layout exactly. They display all the data in a table, for example, even if the table spans multiple pages.
```
Comment (maggiesMSFT) [inline powerbi-docs/paginated-reports/paginated-reports-report-builder-power-bi.md:3]:
```suggestion
description: Paginated reports are now available in the Power BI service. They have long been the standard report format in SQL Server Reporting Services. These reports can be printed or shared. You can control the report layout exactly. They display all the data in a table, for example, even if the table spans multiple pages.  
```

**Comment by PRMerger18** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by maggiesMSFT** [comment]
Thanks, @gsuttie, for your update. The original sentence was grammatically correct, but you're the 2nd person who was troubled by it. I rewrote it. 
Thanks, @ktoliver, for asking me. I'm puzzled how it got to you before it came to me.

**Comment by maggiesMSFT** [comment]
#sign-off

**Comment by ktoliver** [comment]
> Thanks, @ktoliver, for asking me. I'm puzzled how it got to you before it came to me.

@maggiesMSFT PR reviewers are asked to approve and merge PRs in the public repo that submit editorial changes or changes that we can verify (that is, to triage all PRs in the public repos). We do a couple sweeps each weekday for PRs like this and for approved PRs that are ready to merge.

**Comment by maggiesMSFT** [comment]
@ktoliver -- oh, didn't know that. OK, thanks!

**Pull request merged**